src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,19 @@
+package commands;
+
+import core.Ui;
+import data.Storage;
+import tasks.TaskList;
+
+//CHECKSTYLE.OFF: MissingJavadocType
+public class AddCommand extends Command {
+    private String input;
+
+    public AddCommand(String input) {
```

Comment:
Should there be a descriptive header comment for the public method here?

Reply:
Yes!